DataStructures/LinkedList/LinkedList.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+    Author:
+    Tucker J. Polomik (https://github.com/TheNergaL)
+    Name of algorithm:
+    Singly Linked-list
+    Brief description:
+    List comprised of nodes which hold a value
+    and point to the next value element in the list.
+    If there exists no next element, it points to null.
+*/
+
+#ifndef LINKED_LIST_HPP
+#define LINKED_LIST_HPP
+
+template<class T>
+struct Node
+{
+  T value;
+  Node<T> *nextnode;

```

Comment:
Rename `nextnode` to either `next` or `next_node`.
Also, indentation is short by 2 spaces.